src/main/java/org/datadog/jmxfetch/App.java

```diff
@@ -81,6 +82,15 @@ public static void main(String[] args) {
             System.exit(1);
         }
 
+        if( config.getAction().equals( AppConfig.ACTION_LIST_JVMS )) {
+            List<com.sun.tools.attach.VirtualMachineDescriptor> descriptors = com.sun.tools.attach.VirtualMachine.list();
+            System.out.println("List of JVMs for user " + System.getProperty("user.name") );
+            for( com.sun.tools.attach.VirtualMachineDescriptor descriptor : descriptors ) {
+                System.out.println( "\tJVM id " + descriptor.id() + ": '" + descriptor.displayName() + "'" );
+            }
+            System.exit(0);
+        }
```

Comment:
This block can potentially lead to `java.lang.NoClassDefFoundError` when no `tools_jar_path` is defined in the YAML configuration file.
```
root@xxxx:~# sudo /etc/init.d/datadog-agent jmx list_jvms
2016-06-17 15:46:39,166 | INFO | dd.collector | jmxfetch(jmxfetch.py:244) | Starting jmxfetch:
2016-06-17 15:46:39,167 | INFO | dd.collector | jmxfetch(jmxfetch.py:292) | Running java -Xms50m -Xmx200m -classpath /opt/datadog-agent/agent/checks/libs/jmxfetch-0.11.0-jar-with-dependencies.jar org.datadog.jmxfetch.App --check jmx.yaml --check_period 15000 --conf_directory /etc/dd-agent/conf.d --log_level INFO --log_location /var/log/datadog/jmxfetch.log --reporter console --status_location /opt/datadog-agent/run/jmx_status.yaml list_jvms
Exception in thread "main" java.lang.NoClassDefFoundError: com/sun/tools/attach/VirtualMachine
	at org.datadog.jmxfetch.App.main(App.java:86)
Caused by: java.lang.ClassNotFoundException: com.sun.tools.attach.VirtualMachine
	at java.net.URLClassLoader$1.run(URLClassLoader.java:366)
	at java.net.URLClassLoader$1.run(URLClassLoader.java:355)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.net.URLClassLoader.findClass(URLClassLoader.java:354)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:425)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:308)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:358)
	... 1 more
```
Can we catch it ad prompt a more comprehensive error message when this happens ?